pokedex/pokedex.lua

```diff
@@ -394,9 +436,31 @@ function M.get_pokemon(pokemon)
 	end
 end
 
+function M.get_pokemon(pokemon, variant)
+	local raw = M.get_pokemon_raw(pokemon)
```

Comment:
There is also an argument for using.
```lua
function M.get_pokemon(pokemon, options)
  local raw = M.get_pokemon_raw(pokemon)
  local variant = options.variant
  ...
end
```

That way if we in the future need add more things we can put it into the options table and not having to update all function signatures.

Reply:
I do not want to do that when there is just 1 optional parameter. When (and if) there become two optional parameters, then I would change this API. Before then, it's just unnecessary speculative generality which makes the API more complex without an extant reason.